### ISSUE TITLE
TypeError: in cglobal, expected Symbol, got JuliaInterpreter.SSAValue

### ISSUE THREAD

**Issue by fredrikekre** [JuliaDebug/JuliaInterpreter.jl#354]
The following code only interprets after running it in compiled mode once:

```julia
using HTTP, JuliaInterpreter
method = "GET"
url = "https://api.github.com/"
headers = Dict("User-Agent" => "Debugger.jl")
```

```julia
julia> @interpret HTTP.request(method, url, headers)
ERROR: TypeError: in cglobal, expected Symbol, got JuliaInterpreter.SSAValue
Stacktrace:
 [1] #evaluate_call_recurse!#48(::Bool, ::typeof(JuliaInterpreter.evaluate_call_recurse!), ::Any, ::Frame, ::Expr) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:213
 [2] evaluate_call_recurse! at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:202 [inlined]
 [3] eval_rhs(::Any, ::Frame, ::Expr) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:379
 [4] step_expr!(::Any, ::Frame, ::Any, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:519
 [5] step_expr!(::Any, ::Frame, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:558
 [6] finish!(::Any, ::Frame, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/commands.jl:14
 [7] finish_and_return! at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/commands.jl:29 [inlined]
 [8] #evaluate_call_recurse!#48(::Bool, ::typeof(JuliaInterpreter.evaluate_call_recurse!), ::Any, ::Frame, ::Expr) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:240
 ... (the last 7 lines are repeated 7 more times)
 [58] evaluate_call_recurse! at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:202 [inlined]
 [59] eval_rhs(::Any, ::Frame, ::Expr) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:379
 [60] step_expr!(::Any, ::Frame, ::Any, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:435
 [61] step_expr!(::Any, ::Frame, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:558
 [62] finish!(::Any, ::Frame, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/commands.jl:14
 [63] finish_and_return! at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/commands.jl:29 [inlined]
 [64] #evaluate_call_recurse!#48(::Bool, ::typeof(JuliaInterpreter.evaluate_call_recurse!), ::Any, ::Frame, ::Expr) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:240
 [65] evaluate_call_recurse! at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:202 [inlined]
 [66] eval_rhs(::Any, ::Frame, ::Expr) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:379
 [67] step_expr!(::Any, ::Frame, ::Any, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:519
 ... (the last 7 lines are repeated 3 more times)
 [89] step_expr!(::Any, ::Frame, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/interpret.jl:558
 [90] finish!(::Any, ::Frame, ::Bool) at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/commands.jl:14
 [91] finish_and_return! at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/commands.jl:29 [inlined]
 [92] finish_and_return! at /home/fredrik/.julia/packages/JuliaInterpreter/ATirK/src/commands.jl:33 [inlined] (repeats 2 times)
```
But running it once in compiled mode fixes the intepreted run:
```julia
HTTP.request(method, url, headers) # -> 200 OK
@interpret HTTP.request(method, url, headers) # -> 200 OK
```

Edit: 
JuliaInterpreter: https://github.com/JuliaDebug/JuliaInterpreter.jl/commit/8578fc07864c1a7e59f795d6f52cddb353988d89
HTTP: https://github.com/JuliaWeb/HTTP.jl/commit/33261a650f5af5eb94e9aeeb12d1450c5481867a

**Comment by KristofferC** [comment]
> But running it once in compiled mode fixes the intepreted run:

Hmm... that's odd.

**Comment by thudjx** [comment]
I got the same error when debugging using Juno, but it disappears if I run the whole script by command line.

**Comment by timholy** [comment]
This is almost fixed by #456, and I know what needs to be done for the rest.
